client/src/app/site/motions/components/category-list/category-list.component.html

```diff
@@ -78,28 +79,28 @@ <h2 translate>Categories</h2>
         </mat-expansion-panel-header>
 
         <!-- Edit form shows during the edit event -->
-        <form [formGroup]='this.updateForm' *ngIf="editId === category.id" (keydown)="keyDownFunction($event, category)">
-            <span translate>Edit category details:</span><br>
+        <form id="updateForm" [formGroup]='updateForm' *ngIf="editId === category.id" (keydown)="keyDownFunction($event, category)">
+            <span translate>Edit category details</span>:<br>
             <mat-form-field>
-                <input formControlName="prefix" matInput placeholder="{{'Prefix' | translate}}">
-                <mat-hint>
+                <input formControlName="prefix" matInput placeholder="{{'Prefix' | translate}}" required>
+                <mat-hint *ngIf="!updateForm.controls.prefix.valid">
                     <span translate>Required</span>
                 </mat-hint>
             </mat-form-field>
             <mat-form-field>
-                <input formControlName="name" matInput placeholder="{{'Name' | translate}}">
-                <mat-hint>
+                <input formControlName="name" matInput placeholder="{{'Name' | translate}}" required>
+                <mat-hint *ngIf="!updateForm.controls.name.valid">
```

Comment:
same as above

Reply:
We do this in more places. Please open an issue for this.

Reply:
Do we have an overall guideline for this?